python/amici/petab_objective.py

```diff
@@ -25,6 +25,7 @@
 logger = get_logger(__name__)
 
 AmiciModel = Union[amici.Model, amici.ModelPtr]
+ExpData = Union[amici.ExpData, amici.ExpDataPtr]
```

Comment:
Maybe we want to define them in `__init__.py`?

Reply:
> Title misleading, as it doesn't cache, but only allows for caching them outside. Otherwise good.

True

> Maybe we want to define them in __init__.py?

I can do that. Only these two or is there more where x and xPtr should be used synonymously?

Reply:
fun fact: it is defined in `__init__.py` already :rofl:

Reply:
Why not just use those then... :D

Reply:
> I can do that. Only these two or is there more where x and xPtr should be used synonymously?

Not aware of any, but possible

Reply:
> Why not just use those then... :D

Done ;)